index.html

```diff
@@ -0,0 +1,108 @@
+<!doctype html>
+<html>
+
+<head>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <link href="https://fonts.cdnfonts.com/css/new-york-extra-large" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-mono" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <meta charset="UTF-8">
+    <title></title>
+</head>
+
+<body class="body">
+    <div >
+        <div class="top">
+            <h1 class="ROSE"><img width="235px" height="30px" src="rose.png" alt=""></h1>
+            <div class="wholeBurger">
+                <div class="topPartOfTheBurger"></div>
+                <div class="middlePartOfTheBurger"></div>
+                <div class="bottomPartOfTheBurger"></div>
+            </div>
+        </div>
+        <div>
+            <img class="topPhoto" width="375px" height="235px"
+                src="imgonline-com-ua-Resize-urNhT9kARoZ94SZ.jpg">
+        </div>
+        <div class="textAfterTop">
+            <div class="aboutBlog">A few words about this blog platform, Ghost, and how this site was made</div>
+            <div class="smallTextAboutBlog">Why Ghost (& Figma) instead of Medium, WordPress or other options?</div>
+        </div>
+        <div class="lineOfFirstPage"></div>
+    </div>
+    <div class="secondPage">
+        <div class="articles">All articles</div>
+        <div>
+            <img class="secondPagePhotosFIRST" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-bAyRCik3GxZyxPP7.jpg" alt="">
+            <div class="OFFF" >Granny gives everyone the finger, and other tips from OFFF Barcelona</div>
+        </div>
+        <div>
+            <img class="secondPagePhotos" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-XOzt5UX5xp2iXq.jpg" alt="">
+            <div class="helloWorld">Hello world, or, in other words, why this blog exists</div>
+        </div>
+        <div>
+            <img class="secondPagePhotos" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-wwIuzf7bbgI.jpg" alt="">
+            <div class="regarding">Here are some things you should know regarding how we work </div>
+        </div>
+        <div>
+            <img class="secondPagePhotos" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-T3EhyoGpwJ.jpg" alt="">
+            <div  class="updatingList">Updating list of 50+ sources on distributed teams,
+                remote work, and how to make it all work better</div>
```

Comment:
I think `OFFF, helloWorld, regarding, updatingList` classes can be replaced with a single class. use the same classname for the elements that look the same on the design